examples/flutter_firebase_login/packages/authentication_repository/pubspec.yaml

```diff
@@ -10,13 +10,14 @@ dependencies:
   cache:
     path: ../cache
   equatable: ^2.0.3
-  firebase_auth: ^4.1.0
-  firebase_auth_platform_interface: ^6.11.0
-  firebase_core: ^2.1.1
-  firebase_core_platform_interface: ^4.5.1
+  firebase_auth: ^3.4.0
+  firebase_core: ^1.17.1  
+  # Unfortunately, a breaking change was introduced in 4.5.2.
+  # https://github.com/firebase/flutterfire/issues/9806
```

Comment:
According to https://github.com/firebase/flutterfire/issues/9806#issuecomment-1319642334 this is due to partial update of dependencies, and can be resolved better that a version pin by running `flutterfire update`